README.md

```diff
@@ -21,6 +21,8 @@ after-work.js is designed to be used as a dependency while developing your code.
 }
 ```
 
+You can also accomplish this by running `npm install --save-dev after-work`
```

Comment:
you know, I tested the npm install and realized that, but somehow managed to not put the .js in there